BE/app/domains/helpers/database_repository.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy.orm import Session
+from app.infrastructure.postgresql.database import session
+
+
+class DatabaseRepository:
+    def __init__(self):
+        self.db: Session = session
+
+    def get_db(self):
```

Comment:
anh @bao-qnguyen97 
To avoid the non-null constraint violation, I create the function here to make the DB rollback when the commit violates the DB constraint (for example: Insert a duplicate email)